Sources/MenuAnimator.swift

```diff
@@ -179,7 +191,10 @@ class MenuInteractiveTransition: NSObject, UIViewControllerInteractiveTransition
     //MARK: - Private methods
     //
     private func createSnapshotView(from: UIView) -> UIView {
-        let snapshotView = from.snapshotView(afterScreenUpdates: true)!
+        let snapshotViewOptional = from.snapshotView(afterScreenUpdates: true)
+        guard let snapshotView = snapshotViewOptional else {
+            fatalError("Invalid snapshot view")
```

Comment:
UI view method returns optional UIView. I don't know specific cases, it's not described in the documentation. I suppose when there are some issues with ui views stack. As a result we will not have any image to display it instead of our view controller. It'll look ugly. I've decided that this is a fatal error.
But we can use some placeholder image (some dark picture). What do you think about it?